system76driver/actions.py

```diff
@@ -1383,3 +1383,10 @@ def iter_lines(self):
     def perform(self):
         content = '\n'.join(self.iter_lines())
         self.atomic_write(content)
+
+class firefox_enablewebrender(FileAction):
+    relpath = ('usr', 'lib', 'firefox', 'defaults', 'pref', 's76-webrender.js')
+    content = 'pref("gfx.webrender.all", true);\n'
```

Comment:
Probably should set layout.frame_rate to 144 as well

Reply:
In that case, would we also remove `layout.frame_rate` from `default_settings`?

Reply:
Maybe so